tests/data/state_estimation/1os2msr-no-angle/params.json

```diff
@@ -6,8 +6,7 @@
     "newton_raphson": {
       "experimental_features": "enabled",
       "fail": {
-        "reason": "Experimental feature under development",
-        "raises": "PowerGridError"
+        "reason": "Experimental feature under development"
```

Comment:
FYI: @nitbharambe and I agreed that the current implementation is good enough to already review, refactor and merge. Some of the NRSE validation cases were fixed.

A follow-up PR will fix the remaining `xfail`s like this one. That also means that this NRSE implementation is not complete. This is OK because of the `experimental_features` flag that we implemented in #471 : a partial implementation exists but it is still `experimental` and not `stable`.